platform/lib/routers/pages.js

```diff
@@ -165,7 +165,7 @@ pages.get('/*', async (req, res, next) => {
   // Let known file extensions automatically fallthrough as if they could not
```

Comment:
Following the `growXml` naming scheme, it would make sense to rename this file into `growPages`.